mizar/obj/endpoint.py

```diff
@@ -296,6 +315,9 @@ def get_eptype(self):
     def get_mac(self):
         return str(self.mac)
 
+    def get_vni(self):
```

Comment:
there is a  function called get_tunnel_id that does the same thing. we dont need this

Reply:
Fixed